coderd/reaper/detector.go

```diff
@@ -25,6 +25,10 @@ const (
 	// before it is considered hung.
 	HungJobDuration = 5 * time.Minute
 
+	// NotStartedTimeElapsed is the duration of time since the last update to a job
+	// before it is considered hung.
+	NotStartedTimeElapsed = 30 * time.Minute
```

Comment:
"not started", "notstarted" should probably just be "pending" throughout this file